content/blog/2020-volunteering.md

```diff
@@ -0,0 +1,99 @@
+Title: Making PyCon India even more amazing with your help
+Date: 2020-06-30 22:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-call-for-volunteers
+
+Have you ever wished you could contribute or give back to the Python Community? Well, here’s an awesome opportunity for you to do it at PyCon. 
+
+[PyCon India](https://in.pycon.org/2020/) is a grass-roots, community-driven event. What that means is that various communities across the nation and people that love python come together and help in every way they can, to run the event. The conference is operated entirely by volunteers from planning it, finding the speakers, to handling the event successfully.
+
+<!-- PELICAN_END_SUMMARY -->
+
+## What you need to know. 
+
+Before you embark on a journey helping the community make the conference even more amazing, there are a few things you should know:
+
+* There is a lot of work that goes into a conference, and all of it is done by the volunteers. So, naturally, volunteering requires a lot of time and effort, and some activities like handling social media and registrations, a lot more than others. So, make sure you can allocate sufficient time to your volunteering efforts. 

```

Comment:
We actually don't need any help with registrations. 

 handling social media --> Running outreach and social media, building content, finding speakers